cypress/e2e/basic-starter.cy.ts

```diff
@@ -1,5 +1,21 @@
+beforeEach(() => {
+  cy.visit('/');
+});
+
 describe('Basic Starter', () => {
   it('has a navigation menu', () => {
-    cy.visit('/').get('header nav').should('exist.and.be.visible');
+    cy.get('header nav').should('exist.and.be.visible');
+  });
+
+  it('has working nav menu links', () => {
+    const links = ['Home', 'Articles', 'Events', 'People', 'Places'];
+    links.forEach((link) => {
+      cy.get(`[data-cy="${link}"]`)
```

Comment:
Could we use contains here instead of the custom data attribute https://docs.cypress.io/api/commands/contains#Selector?

Reply:
I updated to use contains instead but my initial reasoning to use data-attributes was from reading this
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements